### DISCUSSION TITLE
Make "quarto preview" work in a project with multiple qmd files

### DISCUSSION THREAD

**Discussion by magical-heyrovsky** [quarto-dev/quarto-cli#4049] [Q&A]
**I know this is probably very specific. I read through most of the documentation but maybe I missed something and this is already possible.**

For my quarto template (for papers, etc) I use one qmd file for every chapter and the output is always just a PDF. The file structure looks like this:
```
.
├── document
│   ├── chapter01.qmd
│   └── chapter02.qmd
├── main.qmd
├── _quarto.yml
├── references.bib
└── references.qmd
```

**_quarto.yml**:
```
project:
  render:
    - main.qmd
```

The **main.qmd** just boostraps everything together:

```
\thispagestyle{empty}
{{< pagebreak >}}

\tableofcontents
\listoffigures
\listoftables
{{< pagebreak >}}

{{< include document/chapter01.qmd >}}
{{< include document/chapter02.qmd >}}
<!-- Add more chapters here -->

{{< pagebreak >}}
{{< include references.qmd >}}
```

This gives me full control of the structure and the result (much more than the "book" feature it seems). So when I run `quarto render` everything works as expected, I get one PDF that contains every chapter. But `quarto preview` (the "live reload") does not work good in this situation because from what I can tell, `preview` watches each file separately. So for example when I change something in _chapter01.qmd_ it will render just that one chapter.

Using Linux, I currently solve this problem with a workaround using [entr](https://github.com/clibs/entr):

`ls **/*.qmd | entr quarto render`

This will run `quarto render` every time **any** .qmd file changes. Pretty neat, problem solved. But it would be cool to have that feature out of the box, so when I work together with Windows people they could do the same.

**Comment by winniehell** [comment]
my workaround is to add

```
  resources:
    - **/*.qmd
```

(https://quarto.org/docs/manuscripts/components.html#including-other-resources)

EDIT: this seems to reliably refresh the page but not rerender the output. :disappointed:

**Comment by winniehell** [comment]
there is a corresponding issue: https://github.com/quarto-dev/quarto-cli/issues/2795

so I think this discussion can be closed
